compile/rust/info.go

```diff
@@ -0,0 +1,69 @@
+/*
+	Copyright 2023 Loophole Labs
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+		   http://www.apache.org/licenses/LICENSE-2.0
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package rust
+
+import "path/filepath"
+
+// SignatureInfo is the import information for a Signature
+type SignatureInfo struct {
+	// Local specifies whether the signature is a locally importable signature
+	Local bool
+
+	// PackageName is the name of the signature package
+	PackageName string
+
+	// ImportPath is the import path for the signature, if the signature is local this will be a path on disk
+	ImportPath string
+
+	// ImportVersion is the version of the signature to import, if the signature is local this will be ignored
+	ImportVersion string
+}
+
+func (s *SignatureInfo) normalize() {
+	if s.Local {
+		if !filepath.IsAbs(s.ImportPath) && !filepath.IsLocal(s.ImportPath) {
+			s.ImportPath = "./" + s.ImportPath
+		}
+	}
+}
+
+// FunctionInfo is the import information for a Function
+type FunctionInfo struct {
+	// PackageName is the name of the function package
+	PackageName string
+
+	// ImportPath is the import path for the function, always a path on disk
+	ImportPath string
+}
+
+func (f *FunctionInfo) normalize() {
+	if !filepath.IsAbs(f.ImportPath) && !filepath.IsLocal(f.ImportPath) {
+		f.ImportPath = "./" + f.ImportPath
+	}
+}
+
+// ExtensionInfo is the import information for an Extension
+type ExtensionInfo struct {
```

Comment:
This ExtensionInfo isn't there in the equivalent /compile/golang/info.go file or the typescript one. Shouldn't it be?